samples/client/petstore/java/jersey2/src/main/java/io/swagger/client/model/Cat.java

```diff
@@ -39,7 +39,7 @@ public Cat declawed(Boolean declawed) {
    * @return declawed
   **/
   @ApiModelProperty(value = "")
-  public Boolean isDeclawed() {
+  public Boolean getDeclawed() {
```

Comment:
@hisener please pull the latest as this should be changed to `isDeclawed` to follow Java boolean getter naming convention.